csrankings-j.csv

```diff
@@ -880,6 +880,7 @@ Jessica K. Hodgins,Carnegie Mellon University,http://www.cs.cmu.edu/~jkh,NOSCHOL
 Jessica Lin 0001,George Mason University,http://cs.gmu.edu/~jessica,tRHOWkcAAAAJ
 Jessica Ouyang 0001,University of Texas at Dallas,https://personal.utdallas.edu/~jessica.ouyang,fQ6T_eAAAAAJ
 Jessica R. Hullman,Northwestern University,http://faculty.washington.edu/jhullman,OLa9EKsAAAAJ
+Jessica Sorrell,Johns Hopkins University,https://jess-sorrell.github.io/,hBsSfjAAAAAJ

```

Comment:
![Uploading Screen Shot 2024-08-05 at 10.01.50 AM.png…]()

From this page: https://www.cs.jhu.edu/faculty/